sdk/python/feast/client.py

```diff
@@ -189,21 +173,21 @@ def _serving_service(self):
         """
         if not self._serving_service_stub:
             channel = create_grpc_channel(
-                url=self._config.get(CONFIG_SERVING_URL_KEY),
-                enable_ssl=self._config.getboolean(CONFIG_SERVING_ENABLE_SSL_KEY),
-                enable_auth=self._config.getboolean(CONFIG_ENABLE_AUTH_KEY),
+                url=self._config.get(ConfigOptions.SERVING_URL),
+                enable_ssl=self._config.getboolean(ConfigOptions.SERVING_ENABLE_SSL),
+                enable_auth=self._config.getboolean(ConfigOptions.ENABLE_AUTH),
                 ssl_server_cert_path=self._config.get(
-                    CONFIG_SERVING_SERVER_SSL_CERT_KEY
+                    ConfigOptions.SERVING_SERVER_SSL_CERT
                 ),
                 auth_metadata_plugin=self._auth_metadata,
-                timeout=self._config.getint(CONFIG_GRPC_CONNECTION_TIMEOUT_DEFAULT_KEY),
+                timeout=self._config.getint(ConfigOptions.GRPC_CONNECTION_TIMEOUT),
             )
             self._serving_service_stub = ServingServiceStub(channel)
         return self._serving_service_stub
 
     @property
     def _use_job_service(self) -> bool:
-        return self._config.exists(CONFIG_JOB_SERVICE_URL_KEY)
+        return ConfigOptions.JOB_SERVICE_URL is not None
```

Comment:
wait, shouldnt this be using a getter?

Reply:
should be `self._config.get(ConfigOptions.JOB_SERVICE_URL) is not None`

Reply:
My bad, typo.